kctf-operator/pkg/resources/external-dns.go

```diff
@@ -67,24 +72,58 @@ func NewExternalDnsDeployment() runtime.Object {
 				},
 				Spec: corev1.PodSpec{
 					ServiceAccountName: "external-dns-sa",
-					Containers: []corev1.Container{{
-						Image: "us.gcr.io/k8s-artifacts-prod/external-dns/external-dns:v0.7.2",
-						Name:  "external-dns",
-						Env: []corev1.EnvVar{{
-							Name: "DOMAIN_NAME",
-							ValueFrom: &corev1.EnvVarSource{
-								ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
-									LocalObjectReference: corev1.LocalObjectReference{
-										Name: "external-dns",
+					Containers: []corev1.Container{
+						{
+							Image: "us.gcr.io/k8s-artifacts-prod/external-dns/external-dns:v0.7.2",
+							Name:  "external-dns",
+							Env: []corev1.EnvVar{{
+								Name: "DOMAIN_NAME",
+								ValueFrom: &corev1.EnvVarSource{
+									ConfigMapKeyRef: &corev1.ConfigMapKeySelector{
+										LocalObjectReference: corev1.LocalObjectReference{
+											Name: "external-dns",
+										},
+										Key: "DOMAIN_NAME",
+									},
+								},
+							}},
+							Args: []string{"--log-level=debug", "--source=service", "--source=ingress",
+								"--provider=google", "--domain-filter=$(DOMAIN_NAME)", "--registry=txt",
+								"--txt-owner-id=kctf-cloud-dns"},
+						},
+						{
+							Image: "eu.gcr.io/sdcpocs1/certbot",
```

Comment:
this will be changed in the next PR, after I add the docker-image to the workflow

Reply:
why not update the workflow as part of this PR?

Reply:
I want to experiment with it a bit. Maybe I can extract this information from config, rather than sed-ing the go code.

I don't really know of a good way to play with workflows short of just commiting things directly to github :-P

The code is currently not enabled (the PROD env variable is not set, so the certificates aren't being issued), if you were worried about that (also, we probably wanna keep it that way for the workflow tests)

Reply:
I'll also fix GCSFUSE as that is doing `apt install` at bootstrap, which seems suboptimal